app/bundles/LeadBundle/Tests/Entity/ImportTest.php

```diff
@@ -162,10 +162,10 @@ public function testGetRunTimeSeconds()
 
         $this->assertSame(0, $import->getRunTimeSeconds());
 
-        $this->fakeImportStartDate($import, 600);
-
         $import->end(false);
```

Comment:
Put a small delay before here in the original code, e.g.:
```
usleep(100000);
```
and then run the test and you get about 10% fails.

With just the natural compute time delay it is a bit difficult to get the "random" fail.